client/blocks/inline-help/popover.jsx

```diff
@@ -249,6 +250,7 @@ class InlineHelpPopover extends Component {
 				position="top left"
 				context={ this.props.context }
 				className={ classNames( 'inline-help__popover', popoverClasses ) }
+				ignoreContext={ SupportArticleDialog }
```

Comment:
I thought this was working but it's just breaking the component and causing all clicks to not close the FAB. 😅